.github/workflows/broken-linkcheck.yml

```
@@ -0,0 +1,37 @@
+name: Linkcheck
+on:
+    workflow_dispatch: # Enable manual triggering
+    #schedule:
+    # Run monthly on the 1st day of the month
+        #- cron: '0 0 1 * *'
+    pull_request:
+      paths:
+        - ".github/workflows/broken-linkcheck.yml"
+        - "doc/make.py"
+jobs:
+    linkcheck:
+        runs-on: ubuntu-latest
+        defaults:
+          run:
+                shell: bash -el {0}
+
+        steps:
+        - name: Checkout
+          uses: actions/checkout@v4
+          with:
+                fetch-depth: 0
+
+        - name: Set up Conda
+          uses: ./.github/actions/setup-conda
+
+        - name: Build Pandas
+          uses: ./.github/actions/build_pandas
+
+        - name: Run linkcheck script
+          working-directory: ./doc
+          run: python make.py linkcheck | tee linkcheck.txt
```

My suggestion actually broke the check. Since we are piping to `tee`, the non-zero exit code, i.e. the failure of the linkcheck, is lost.

```suggestion
          shell: bash
          run: |
            set -o pipefail
            python make.py linkcheck | tee linkcheck.txt
```

---

Could you please explain this change in these `32-35` lines ?